DS/DSassignment1/DoubleeNode.java

```diff
@@ -0,0 +1,259 @@
+
+public class DoubleeNode<T> 
+{
+	
+	T nodeValue;
+
+	DoubleeNode<T> start;
```

Comment:
ACTION: Should each and every "DoubleeNode" have a 'start' member? Or should there be only one 'start' in a doubly list?